packages/sanity/src/core/studio/components/navbar/resources/helper-functions/hooks.ts

```diff
@@ -0,0 +1,18 @@
+import {useMemo} from 'react'
+import {LoadableState, useLoadable} from '../../../../../util'
+import {useClient} from '../../../../../hooks'
+import {ResourcesResponse} from './types'
+import {getHelpResources} from './helpResources'
+
+/**
+ * Fetch help resources (content for the navbar help menu) from `/help`.
+ *
+ * @internal
```

Comment:
`@hidden` tag is also need to make sure it gets hidden from the docs

```suggestion
 * @internal
 * @hidden
```